go/ql/lib/semmle/go/controlflow/ControlFlowGraphImpl.qll

```diff
@@ -623,13 +623,15 @@ module CFG {
       not cmpl.isNormal()
     }
 
-    predicate succ(ControlFlow::Node pred, ControlFlow::Node succ) {
+    predicate succSimple(ControlFlow::Node pred, ControlFlow::Node succ) {
```

Comment:
Can we come up with a name that say what's different about it? `succWithoutUnwindEdges` perhaps?

Reply:
`succWithoutExecutingDeferredFunctions`?

Reply:
Ah yes, a private `succ0` seems fine, with qldoc noting what's missing. Assuming there's no reason someone outside this module would want succ-but-not-unwind-sequences.

Reply:
I can't see why they would want that.